src/HuC3.md

```diff
@@ -0,0 +1,171 @@
+
+Contains the first 16 KiB of the ROM.
+
+### 4000–7FFF – ROM Bank 00–7F (Read-Only)
+
+This area may contain any of the further 16 KiB banks of the ROM. It is
+unknown whether bank $00 can be selected here.
+
+### A000–BFFF – RAM Bank 00–03, or RTC/IR register (Read/Write)
+
+Depending on the current register selection and RAM Bank Number (see
+below), this memory space is used to access an 8 KiB external RAM Bank,
+or a single I/O Register.
+
+
+## Memory Control Registers
+
+### 0000–1FFF – RAM/RTC/IR Select (Write-Only)
+
+Writing to this register maps cart RAM, RTC registers or IR registers
+into memory at A000–BFFF. Only the lower 4 bits are significant.
+
+Value | Feature
+------|-----------------------------
+  $0  | Cart RAM (read-only)
+  $A  | Cart RAM (read/write)
+  $B  | RTC command/argument (write)
+  $C  | RTC command/response (read)
+  $D  | RTC semaphore (read/write)
+  $E  | IR (read/write)
+
+If any other value is written, reads from A000–BFFF return open bus
+values (often $FF, but not guaranteed), and writes are ignored.
+
+### 2000–3FFF – ROM Bank Number (Write-Only)
+
+HuC-3 can accept a 7-bit bank number here.
+
+### 4000–5FFF – RAM Bank Select (Write-Only)
+
+HuC-3 can accept a bank number of at least 2 bits here.
+
+### 6000–7FFF – Nothing? (Write-Only)
+
+Games write $01 here on startup, but it doesn\'t seem to have any
+observable effect.
+
+
+## I/O Registers
+
+These registers are accessed by reading or writing int the A000–BFFF
+range after setting the RAM/RTC/IR Select register. Address lines
+A12–A0 are not connected to the HuC-3 chip, so the offset within the
+range is ignored. Data line D7 is not connected to the HuC-3 chip, so
+the most significant bit of reads always returns an open bus value
+(usually high), and the most significant bit of writes is always
+ignored.
+
+The RTC MCU communication protocol is described below.
+
+### B – RTC Command/Argument (Write)
+
+The value written consists of a command in bits 6-4, and an argument
+value in bits 4–0. For example the value $62 is command $6 with argument
+value $2. Writing to this register just sets the values in the mailbox
+registers – it does not cause the command to be executed.
+
+### C – RTC Command/Response (Read)
+
+When read, bits 6–4 return the last command written to register $B, and
+bits 3–0 contain the result from the last command executed.
+
+### D – RTC Semaphore (Read/Write)
+
+When reading, the least significant bit is high when the RTC MCU is
+ready to receive a command, or low when the RTC MCU is busy.
+
+Writing with the least significant bit clear requests that the RTC MCU
+execute the last command written to register $B.
+
+### E – IR (Read/Write)
+
+Similar to the equivalent register of the HuC-1. The least significant
+bit is used for infrared communication.
+
+
+## RTC Communication Protocol
+
+The HuC-3 chip provides read and write access to a 256-nybble window
+into the RTC MCU\'s internal memory. Multi-nybble values are stored
```

Comment:
Is the backslash necessary?

Reply:
I saw it in use in other markdown documents in your repository (e.g. HuC1.md).  Presumably it’s to prevent automatic conversion of foot/inch marks to typographical quotes.  Should I just use typographical quote characters?

Reply:
Yeah, those should be more appropriate here. I think we have some backslashes left over from the MediaWiki conversion, but in general, I'd rather not have them if they aren't necessary.

Reply:
OK, I’ll fix it.